proto/frequenz/api/common/v1/metrics/metric_sample.proto

```diff
@@ -63,6 +63,17 @@ message MetricValueVariant {
 }
 
 // List of supported metrics.
+
+> [!NOTE]
+> AC energy metrics information:
+> * This energy metric is reported directly from the component, and not a
+>  result of aggregations in our systems. If a component does not have this
+>  metric, this field cannot be populated.
+>  * Components that provide energy metrics reset this metric from time to
+>  time. This behaviour is specific to each component model. E.g., some
+>  components reset it on UTC 00:00:00.
+>  * This energy metric does not specify the timestamp since when the energy
+>  was being accumulated, and therefore can be inconsistent.
```

Comment:
The comment looks incorrect .. these should start wist `//`, no?

Maybe better to use the MkDocs format here. Here is an example:

```
// !!! note
//     This is a note.
//     This is the second line.
```

Reply:
Ah sorry, I thought it was suppose to look like in the markup docs (but maybe I looked at the wrong one).

Reply:
Sounds good :-)